.github/workflows/develop.yml

```
@@ -0,0 +1,48 @@
+name: Build and Deploy Jenkins
+on:
+  push:
+    branches:
+      - develop
+jobs:
+  build:
+      name: Build and push new islasgeci/jenkins image
+      runs-on: ubuntu-latest
+      steps:
+      - name: Copia repositorio
+        uses: actions/checkout@v2
+      - name: Construye imagen
+        run: docker build --tag islasgeci/jenkins:latest --tag islasgeci/jenkins:${GITHUB_SHA:0:4} .
+      - name: Docker log-in
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+        run: echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin
```

Por favor agrega la versión larga de la opción. Creo que es esta:
```suggestion
        run: echo "${DOCKER_PASSWORD}" | docker login --username "${DOCKER_USERNAME}" --password-stdin
```